test/e2e/modal.test.js

```diff
@@ -102,29 +104,29 @@ describe('vl-modal', async () => {
     it('als gebruiker kan ik meteen typen in het input-veld in de modal in Safari', async () => {
         const modal = await vlModalPage.getModalSafari();
         await vlModalPage.openModalSafari();
-        const content = await modal.getContent();
-        await assert.eventually.isTrue(content.hasFocus());
+        const slotElements = await modal.getContentSlotElements();
+        const input = await new VlInputField(driver, slotElements[0]);
+        await assert.eventually.isTrue(input.hasFocus());
         await modal.close();
     });
 
-    it('als gebruiker zie ik een verticale scrollbar als er te veel tekst in de modal staat', async() => {
+    it('als gebruiker kan ik verticaal scrollen als er te veel tekst in de modal staat', async() => {
         await vlModalPage.openModalMetVeelTekst();
         const modal = await vlModalPage.getModalMetVeelTekst();
         await modal.scrollToTop();
```

Comment:
dat gaat mis dan als er geen scroll is ?

Reply:
De initiële bug was dat er geen scroll was, dus als deze test faalt is er een probleem.

Reply:
als scrollToTop geen error gooit als er geen scroll is, gaat de test altijd groen zijn

Reply:
De test zal inderdaad niet rood zijn omdat de scroll programmatisch gebeurt. Ik denk dat de functionaliteit goed genoeg gedekt is via de unit test `laat alleen overflow toe bij dialog indien het allow-overflow attribuut aanwezig is` en deze e2e test puur het gebruik van de scrollToTop demonstreert, maar verder zinloos is. Deze functie aanroepen verzekert dat er niets fout loopt in het page object, maar dekt inderdaad de bug niet.